sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/slot/GatewaySlotChainBuilder.java

```diff
@@ -31,8 +31,13 @@
 /**
  * @author Eric Zhao
  * @since 1.6.1
+ *
+ * @deprecated since 1.7.2, we can use @SpiOrder(-4000) to adjust the order of slots,
+ * this class is reserved for compatibility with older versions.
+ * @see {@link GatewayFlowSlot}
  */
-public class GatewaySlotChainBuilder implements SlotChainBuilder {
+@Deprecated
+public class GatewaySlotChainBuilder extends DefaultSlotChainBuilder {
```

Comment:
Oh if you take my advice keeping an empty legacy one you should make it a real EMPTY one.
According to the new `DefaultSlotChainBuilder` i think there is no need keeping overriding of `build()` here, right?

Reply:
I'm sorry for misunderstanding the point.It has been fixed, please check.